src/rules/no-extended-unicode-identifiers.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = {
+	meta: {
+		type: 'problem',
+		docs: {
+			description: 'Prohibits non-BMP characters in identifiers, for Safari 11-13 compatibility'
```

Comment:
Nit: Can you add a full stop at the end of both of these? (I'd do an edit, but it needs the markdown file to be regenerated and I'd rather not make a mess of the branch.)